metadata/metadata.yml

```diff
@@ -0,0 +1,2 @@
+metadata_version: 1
+name: aind-smartspim-classification-test
```

Comment:
We should change this name to aind-smartspim-cell-classification

Reply:
fixed